repository-data/application/src/main/resources/hcm-content/content/documents/administration/publication-system/labels.yaml

```diff
@@ -27,7 +27,7 @@
     - Publication
     - Data set
     - Series / Collection
-    - Search Clinical Indicators
+    - Search in Indicators
```

Comment:
Don't bother doing label changes on here just change them on the environments as this will have no effect except for local environments

Reply:
ok